src/components/TransactionBlock/KlimaStakingTransactionBlock.tsx

```diff
@@ -320,6 +323,26 @@ const KlimaStakingTransactionBlock = ({
     receiverAddress,
   ])
 
+  const getBestRouteIndex = (items: BridgingQuote[]) => {
```

Comment:
not sure why we return index here and not just best `BridgingQuote`?

Reply:
also make sure u define in comments how best offer is selected

Reply:
@MayukhSamadder2000 not sure why we return index here and not just best `BridgingQuote`?